### PR TITLE
feat(std):Eslint airbnb with reduce error

### PR DESCRIPTION
**Pull request by chintan9**

### CODE REVIEW

**Comment by chompomonim** [comment]
Why you wanna push to use semicolon in this starter kit?

File: .eslintrc [hunk 1]
```diff
@@ -1,40 +1,60 @@
...
+    "jsx-quotes": [
+      2,
+      "prefer-single"
+    ],
+    "max-len": [
+      2,
+      120,
+      2
+    ],
+    "semi": [
```
Comment (chintan9) [inline .eslintrc]: I will remove semi colon

**Comment by davezuko** [comment]
Not sure what the objective of this PR is. If you'd like to resubmit with some details I'd be happy to discuss.

**Comment by chintan9** [comment]
I wanna put airbnb style guide for starter kit

**Pull request closed without merging**